src/main/java/org/syngenta/trainings/websockets/websocket/SensorDataSocketHandler.java

```diff
@@ -17,6 +19,7 @@
 import java.util.concurrent.TimeUnit;
 
 @Slf4j
+@Service
```

Comment:
Is it really a service? 
We can create new separate component based on controller.
Something like: 
@Target({ElementType.TYPE})
@Retention(RetentionPolicy.RUNTIME)
@Documented
@Controller
public @interface WebSocketHandler {

    @AliasFor(annotation = Controller.class)
    String value() default "";
    
}

Reply:
`Is it really a service?`
yes, it is service

Reply:
`We can create new separate component based on controller.`

Reply:
It was just an idea.
Probably, it will be a lot of such WS handlers and it may be useful to mark them.

Reply:
If there are multiple web-socket handlers then we can autowire to `List<TextWebSocketHandler>` in configuration